### ISSUE TITLE
missing /dev/mem results in incorrect grains

### ISSUE THREAD

**Issue by sjorge** [saltstack/salt#36907]
### Description of Issue/Question
```yaml
    productname:
        /dev/mem: No such file or directory
    biosreleasedate:
        /dev/mem: No such file or directory
    biosversion:
        /dev/mem: No such file or directory
    manufacturer:
        /dev/mem: No such file or directory
```

Probably better to have the grain emtpy... or not exist when /dev/mem is not present.

### Setup
Ubuntu under SmartOS as a LX zone

**Comment by sjorge** [comment]
Fixed with a PR #36909

**Comment by Ch3LL** [comment]
Perfect! thanks for the PR. Feel free to close once your PR has been merged or we can. :+1:
